extensions/sql-migration/src/api/azure.ts

```diff
@@ -712,7 +712,7 @@ export async function validateIrDatabaseMigrationSettings(
 			userName: migration._sqlServerUsername,
 			password: migration._sqlServerPassword,
 			trustServerCertificate: trustServerCertificate,
-			encryptConnection: true,
+			encryptConnection: false,
```

Comment:
I don't think you should be setting this to 'false' directly here. 

Wherever the user is providing the 'sourceSqlConnection' properties - the user should specify these properties: 'encrypt' and 'trustServerCertificate' as well.

Reply:
@cheenamalhotra, for this specifically, we discussed offline and this is the temporary mitigation for now, as our SQL Migration RP is not properly respecting the value of this property anyways. But agreed the long term solution is to respect the user input

Reply:
In either case, you should not disable encryption. Enabling 'trustServerCertificate' will bypass certificate validation so that connections can still be encrypted without failure.